home/views.py

```diff
@@ -7,3 +8,30 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+def displayQuestion(request, **kwargs):
+    question = Question.objects.all().get(id=kwargs['pk'])
+    sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
+    context = create_display_question_context(question, sortAnsBy)
+    return render(request, 'home/question_detail.html', context)
+
+
+def create_display_question_context(question, sortAnsBy):
+    context = {}
+    context['question'] = question
+
+    answers = question.get_answers_feed(sortAnsBy)
+    context['answers'] = answers
+
+    # Get count of answers string
+    answersCount = answers.count()
+    context['answersCountStr'] = str(answersCount) + ' Answer' + ('s' if answersCount > 1 else '')
```

Comment:
This can be a custom template function, though IIRC the templating engine already has nice functions for dealing with plural v.s. sigular value titles.

Reply:
Ok, fixed